src/main/webapp/jsp/list-employees.jsp

```diff
@@ -7,7 +7,7 @@
         <link rel="stylesheet" href="css/bootstrap.min.css" >   		
         <script src="js/jquery.min.js"></script>
         <script src="js/bootstrap.min.js" ></script>        
-	    <title>Employees Portal</title>
+	    <title>HR Portal</title>
```

Comment:
Reviewer : Kindly convert title in UPPERCASE

Reply:
Yes sir we can see the comment

Reply:
DONE! Kindly accept the changes